### ISSUE TITLE
New module dedicated to Projects (Memex)

### ISSUE THREAD

**Issue by a-a-ron** [githubpartners/microsoft-learn#38]
We need to create a dedicated module for Projects.

**Timeline: TBD**

[Projects](https://github.com/features/issues) (also known as Memex, [Issues vNext](https://thehub.github.com/initiatives/github-issues-vnext/), Projects V2, Projects Beta) is a tool for project management built directly into GitHub using [React](https://reactjs.org/).

Note: the name "Projects" is also shared by [another product of the same name](https://docs.github.com/en/issues/organizing-your-work-with-project-boards/managing-project-boards/about-project-boards), which the Memex project has now replaced, and is now called "Projects (classic)."

## Resources

- [Memex repository](https://github.com/github/memex)
- [Public documentation](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects)
- [Memex marketing page](https://github.com/features/issues)

**Comment by camihmerhar** [comment]
Howdy @a-a-ron!

Here's an outline of how I envision the module with links to where I plan to pull the content from. Let me know your thoughts! 

## Introduction
- About Projects - GitHub Docs
- GitHub Issues - Project planning for developers
- Pre-requisites
   - A GitHub Account
   - Access to a GitHub organization
 ## How to build an organization project
- [Creating an organization project](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/quickstart-for-projects#creating-an-organization-project)
- [Setting your project description and readme](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/quickstart-for-projects#setting-your-project-description-and-readme)
- [Adding issues to your project](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/quickstart-for-projects#adding-issues-to-your-project)
- [Adding drafted issues to your project](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/quickstart-for-projects#adding-draft-issues-to-your-project)
- [Configure build In Automation](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/quickstart-for-projects#configure-built-in-automation)
## Organizing your project 
- [Adding an iteration field](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/quickstart-for-projects#adding-an-iteration-field)
- [Creating a field to track priority](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/quickstart-for-projects#creating-a-field-to-track-priority)
- [Adding a board layout](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/quickstart-for-projects#adding-a-board-layout)
## Exercise 
## Knowledge Check
## Summary

**Comment by rmallorybpc** [comment]
@camihmerhar 
Intro suggestions
- [ ] I'd change, Luckily GitHub with Projects, to something like, Luckily using GitHub with Projects, , GitHub with Projects (Projects), or, GitHub and Projects. The way I'm reading the current sentence is that the name of the product is GitHub with Projects.
- [ ] I'd add the third content unit (when to use) to the learning objectives.
- [ ] The creating an organization section could be a good spot for an image. Or really any of the section with step lists.

**Comment by rmallorybpc** [comment]
@camihmerhar Unit 2-how-to-create-a-project.md suggestions
- [ ] Creating an organization project "4." needs a period at the end.
- [ ] The interaction UI should be bold, example below: First, in the top right corner of GitHub.com, click **your profile photo**, then click **Your organizations**.
- [ ] Also, the UI should be the same upper or lower case as the UI. 2. In the menu, click settings. Settings may be lower case in the UI, so this may be a simple double check. This is one that stood out to me without looking at the UI, so I could be off in this specific case.
- [ ] The sections, Adding, and, Searching, some periods are needed in the numbered lists.
- [ ] In the transition statement: I think they want us to stay away from the use of "we". This could be something to double check with Aaron.

**Comment by rmallorybpc** [comment]
3-how-{product}-works.md suggestions
- [ ] Add periods to the numbered lists where there are no periods. Across unit.
- [ ] A UI image could be good to add to one of the bullet lists under, How to create a field to track and group by priority.
- [ ] Apply **bold** to the clickable text. Example: 2. Then click, **New Field**. Across unit.
- [ ] Apply ***italics*** to the field names. Example: this in the field row titled ***Priority***. Across unit.
- [ ] Think about splitting this sentence into more than one.
      To do this, select the issue or pull request you want to classify, this in the field row titled Priority, in the drop down field select the priority you want to choose for this particular issue or pull request.
- [ ] Think about adding some more context to the beginning of each section, before getting right into the bullet lists. Also, a good entry if you want to find some more images.
      - This is a [good blog entry](https://github.blog/2022-07-27-planning-next-to-your-code-github-projects-is-now-generally-available/) for some more context information.
      - [Another one for context](https://github.blog/2022-07-21-tips-tricks-for-using-github-projects-for-personal-productivity/).
- [ ] Add a transition sentence to the knowledge check.

**Comment by rmallorybpc** [comment]
The knowledge check questions look good.

**Comment by camihmerhar** [comment]
Thanks for the feedback @rmallorybpc! I might be misremembering, but I definitely saw in the [Author Pack](https://github.github.com/ms-learn-author-pack/#/writing/writing-style-guide?id=titles-and-headers), like you mentioned, we should be ***italicizing*** field names. But now I can't find -- did they just update it and I assume we should follow the new guidelines to just **bold** UI elements?

**Comment by rmallorybpc** [comment]
@camihmerhar yeah, I was referencing the [markdown styling information](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).
@a-a-ron thoughts?
